### ISSUE TITLE
PE feedback regarding duplicates 💡

### ISSUE THREAD

**Issue by Smanmos** [nus-cs2103-AY2122S1/forum#425]
From my observation, it appears that CATcher will display an issue as a duplicate only if it is marked as a duplicate of something else. 

Since issues marked as duplicate are essentially considered by the dev team to be the same, I think that an issue’s maker should also be informed if another issue has been labelled a duplicate of it. Furthermore, if an issue has 2 or more other issues marked as duplicates, each tester should be allowed to argue against each other issue being marked as a duplicate. 

While not something that can be changed at this point, it would be useful for future iterations of this module.

**Comment by damithc** [comment]
Fair point @kawaiigabrielneko We'll consider it in the next semester, weighing the benefit against the cost of increased workload (more things to check for testers, more disputes to resolve for tutors).
